README.md

```diff
@@ -24,3 +24,47 @@
 
 - [x] ARYAN: I will prepare the dataset -- downloading + loading them up in notebook
 - [x] MKO: I will add some of the suggested topics (datasets) -- downloading + loading them up in notebook
+
+## Plan for pp
+
+How to be happier?
+Ask questions from some student's percpective.
+
+## Charcater sugestion
+Forrest Gump (from Forrest Gump):
+Personality: Simple, sincere, and often unintentionally humorous with his straightforward observations.
+Example Questions:
+"Mama always said life is like a box of chocolates, but does more money mean sweeter happiness?"
+"If I keep running, will I find happiness or just get really tired?"
```

Comment:
Unlike money, we don't have a running dataset.

Reply:
> Unlike money, we don't have a running dataset.

not everyone here has money :(

Reply:
ok guys forest gump was just a random suggestion, lets forget about him.

Reply:
I will remove the part